open_fdd/rules/cookbook_catalog.py

```
@@ -0,0 +1,1939 @@
+from typing import Callable
+
+import numpy as np
+import pandas as pd
+
+# ---------------------------------------------------------------------------
+# Shared helpers (mirror cookbook "Setup & shared helpers")
+# ---------------------------------------------------------------------------
+
+
+def norm_cmd(s: pd.Series | None) -> pd.Series:
+    """Normalize a command/position 0–100 → 0–1. Passthrough if already 0–1."""
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)